tests/test_util.ts

```diff
@@ -23,29 +22,25 @@ export class TestSuite {
   }
 
   test(name: string, func: TestFunc): void {
-    this.tests.push({ name, func });
+    Deno.test(`[${this.prefix}] ${name}`, async () => {
```

Comment:
We want to utilize Deno's leaking check feature :smiley:.
So, I run `func` here.